Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1230,6 +1230,20 @@ CSSDPropertyEnabled:
     WebCore:
       default: false
 
+CSSDynamicRangeLimitMixEnabled:
+  type: bool
+  category: css
+  status: unstable
```

Comment:
You can use `testable` to keep the WPT passing, people can also choose to enable it from the Feature Flags too